<ui version="4.0" >
 <class>RunSettings</class>
 <widget class="QWidget" name="RunSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Run Options</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>&amp;Run Target:</string>
       </property>
       <property name="buddy" >
        <cstring>runTarget</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="runTarget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis" >
        <string>A "run target" is a way of grouping settings for running a program.  The "Project Default" target will be used for any targets which do not have individualised configurations.  Create a new run target to define specific settings for a target.</string>
       </property>
       <property name="editable" >
        <bool>false</bool>
       </property>
       <item>
        <property name="text" >
         <string>Project Default</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProjectNameIndicator" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="buttonNewTarget" >
       <property name="text" >
        <string>&amp;New Target..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="buttonDeleteTarget" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Delete Target</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Main Program</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="notelabel" >
        <property name="whatsThis" >
         <string>For Automake and QMake type projects, the proper way to set these options are per target in the &lt;b>Automake Manager&lt;/b> and &lt;b>QMake Manager&lt;/b>, respectively.</string>
        </property>
        <property name="text" >
         <string>Note: These options override target specific settings.</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="mainprogram_label" >
        <property name="text" >
         <string>Executa&amp;ble:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_executable</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KUrlRequester" name="kcfg_executable" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="mainprogram_label_2" >
        <property name="text" >
         <string>Arg&amp;uments:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_arguments</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KLineEdit" name="kcfg_arguments" >
        <property name="toolTip" >
         <string>The command line arguments passed to the main program when run</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="mainprogram_label_3_2" >
        <property name="text" >
         <string>&amp;Working Directory:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>kcfg_workingDirectory</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="KUrlRequester" name="kcfg_workingDirectory" />
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&amp;Environment</string>
        </property>
        <property name="buddy" >
         <cstring>kcfg_environment</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="KDevelop::EnvironmentSelectionWidget" name="kcfg_environment" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="kcfg_autocompile" >
        <property name="toolTip" >
         <string>If the program is not up-to-date with the source code, compile it before starting its execution</string>
        </property>
        <property name="text" >
         <string>Automaticall&amp;y compile before execution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_autoinstall" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="text" >
         <string>&amp;Automatically install before execution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_autosu" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="text" >
         <string>Use &amp;super user when installing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_startinterminal" >
        <property name="toolTip" >
         <string>Start the main program in an external terminal</string>
        </property>
        <property name="text" >
         <string>Start in e&amp;xternal terminal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>598</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KDevelop::EnvironmentSelectionWidget</class>
   <extends>KComboBox</extends>
   <header>environmentselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_startinterminal</tabstop>
  <tabstop>kcfg_autocompile</tabstop>
 </tabstops>
 <includes>
  <include location="global" >ksqueezedtextlabel.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kurlrequester.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kpushbutton.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kurlrequester.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
